popupbridge-paypaldatacollector-ios.md

```diff
@@ -22,7 +22,7 @@ The call to `+[PPDataCollector clientMetadataID:]` causes device data to be coll
 
 ## Charging a vaulted PayPal account
 
-1. Create a global JavaScript function on your web page to receive the device data object and [provide it to your server](https://developers.braintreepayments.com/guides/paypal/vault/javascript/v3#collecting-device-data), e.g. by injecting it into your form as a hidden input.
+1. Create a global JavaScript function on your web page to receive the device data object and [provide it to your server](https://developer.paypal.com/braintree/docs/guides/paypal/vault/ios/v5#collecting-device-data), e.g. by injecting it into your form as a hidden input.
```

Comment:
Should this be changed to ios? Or should it remain JS? I assume since it's rendering the page in a webview, it should remain JS.

Reply:
Oh, yep you're right 🤦